src/Features/LanguageServer/Protocol/Features/Diagnostics/DiagnosticAnalyzerService.cs

```diff
@@ -199,6 +206,8 @@ public async Task ForceAnalyzeAsync(Solution solution, Action<Project> onProject
             }
         }
 
+        public bool WasForceAnalyzed(ProjectId projectId) => _forceAnalyzedProjectIds.Contains(projectId);
```

Comment:
won't this set just fill up until it contains all projects.  At which point it always returns true?